recipe/meta.yaml

```diff
@@ -16,7 +16,6 @@ build:
   skip: true  # [osx and arm64 and py < 39]
   entry_points:
     - spyder = spyder.app.start:main
-  osx_is_app: true
```

Comment:
Why is this not necessary anymore?

Reply:
Sorry, I added this back in.
Originally, when building the installer, I patched the feedstock in a way that imitated what napari was doing and I just duplicated it here. This may be required for a standard conda installation and I'll verify that it does not impact the conda-based installation.